src/curvefit/models/core_model.py

```diff
@@ -159,3 +186,12 @@ def bounds(self):
         re_bounds = np.repeat(re_bounds[None, :, :], self.data_inputs.num_groups, axis=0)
 
         return np.vstack([fe_bounds, re_bounds.reshape(self.param_set.num_fe * self.data_inputs.num_groups , 2)])
+
+    @property
+    def x_init(self):
+        fe_init = np.array(reduce(iconcat, self.param_set.fe_init, []))
```

Comment:
@jizezhang aren't these already reduced in the `ParameterSet` object?

Reply:
so from `Variable` to `Parameter`, `fe_init` are collected into a list, and then from `Parameter` to `ParameterSet`, they are collected into a list of lists. I was also thinking where to do the flattening. Which do you think would be better, inside `ParameterSet` or here in the `CoreModel` ?

Reply:
Probably in `CoreModel` where it is now!